### ISSUE TITLE
tfdv.validate_tensor_examples()?

### ISSUE THREAD

**Issue by JakeTheWise** [tensorflow/data-validation#106]
I think it would be nice to have a top-level function to check for anomalies in serving data. It could be integrated into `serving_input_receiver_fn`.

Or is there already a best practice for doing this?

**Comment by rmothukuru** [comment]
@JakeTheWise,
 We currently, can check if Serving Data has Anomalies using the below code:

```
serving_stats = tfdv.generate_statistics_from_tfrecord(data_location=serving_data_path)
serving_anomalies = tfdv.validate_statistics(serving_stats, schema)
```
For more information, please refer this [TFDV Page](https://www.tensorflow.org/tfx/data_validation/get_started#schema_environments).

Please let me know if this is what you are looking for, or if your proposal is different. Thanks!

**Comment by JakeTheWise** [comment]
Yes but why do I need to write serving data to tfrecords just to check for anomalies?

**Comment by caveness** [comment]
Hi Jake -- Are you looking to validate your serving examples one at a time?

**Comment by JakeTheWise** [comment]
In the limit, yes. Serving examples could come in one at a time, or in batches. Thus the idea to have a tensor-based example validator, as the serving environment (tensorflow model server) / model export signature function will receive examples as (dictionaries of) tensor objects.

**Comment by bharatjetti** [comment]
Closing this due to inactivity. Please feel free to reopen and post your comments(if you still have queries on this). Thank you!
